### ISSUE TITLE
Better UI needed

### ISSUE THREAD

**Issue by sijan8s3** [sijan8s3/MusicPlayer#2]
better and attractive UI is needed for this app.
feel free to contribute.

**Comment by gargk747** [comment]
Can i work on this issue?

**Comment by sijan8s3** [comment]
@gargk747  yes, of course!
star the repo,
fork it,
and work on it,
then send me a merge request.
I'd be happy to merge it! :)

**Comment by sameershekhar100** [comment]
the app is crashing

**Comment by sijan8s3** [comment]
@MERCUROUS 
this is a bit old repo,
I need to check the code and logs.
I'll let you know when I check and fix it.

**Comment by gargk747** [comment]
I have sent the merge request... please kindly check!

Thanks

**Comment by andrehgdias** [comment]
Could you guys post the images with the changes you've made? I'm just curious and since you all've been working on this I won't disturb you with more versions...

**Comment by ziyadvf** [comment]
Hey there!
You guys looking for a better UI, I could definitely be able to create a better UI.
I would love to join.

**Comment by sijan8s3** [comment]
@andrehgdias  @ziyadfarhan296 
I'd love some contributions to UI and features.
make sure to star the repo.
I'll surely merge your PRs.

**Comment by ziyadvf** [comment]
Yes, I have starred this repo,
Sure thing, excited to be here!

**Comment by ziyadvf** [comment]
So when should we get started?
I am ready to get to work, seems a great team to be in!

**Comment by azzam-pixel** [comment]
Nice

**Comment by sijan8s3** [comment]
@ziyadfarhan296 
everyone's contributing,
if you have any idea on features,
feel free to comment.

**Comment by 12Aivars** [comment]
![20200808_162542](https://user-images.githubusercontent.com/71190041/95197494-5b423100-07e2-11eb-93cd-d074077b795d.jpg)
vis super

**Comment by 12Aivars** [comment]
super puper

**Comment by ziyadvf** [comment]
I have worked on this home screen - simple and basic design, but thoughtful.
![Music](https://user-images.githubusercontent.com/56755783/95202717-5e411f80-07ea-11eb-816c-175a760d8758.png)

**Comment by sijan8s3** [comment]
@ziyadfarhan296 
this is amazing! 
very creative!

**Comment by ziyadvf** [comment]
Yeah, I have also worked on the music playing screen, which has all the music controls and the ability to set which speakers you want to use.
![Music – Music Interaction](https://user-images.githubusercontent.com/56755783/95203221-1d95d600-07eb-11eb-92d1-1855798f0534.png)

**Comment by ziyadvf** [comment]
I am using adobe XD to create these designs, so there are some things I can't post here such as notes, interactions, and other stuff like that.

**Comment by sijan8s3** [comment]
if you've dribble mention the link, I'd love to see more!
you do have a good designing concept.

UI is a major part of development,
which I lack! :D

**Comment by ziyadvf** [comment]
Yes, I will finish up the concept and upload it on Dribbble.

**Comment by ziyadvf** [comment]
There you go, I finished the designs!
Dribbble: https://dribbble.com/shots/14348978-Music-App
Downloadable file:
[UI Images.zip](https://github.com/sijan8s3/MusicPlayer/files/5347328/UI.Images.zip)
